src/sage/schemes/elliptic_curves/hom_composite.py

```diff
@@ -893,3 +894,290 @@ def inseparable_degree(self):
             1331
         """
         return prod(phi.inseparable_degree() for phi in self._phis)
+
+    def is_cyclic(self, _check_j=True, _reduce_kernel=True):
```

Comment:
```suggestion
    def is_cyclic(self, *, _check_j=True, _reduce_kernel=True):
```
(To avoid people accidentally using these internal arguments.)